vast/vast.cpp

```diff
@@ -92,11 +92,13 @@ int main(int argc, char** argv) {
   // Print the configuration file(s) that were loaded.
   if (!cfg.config_file_path.empty())
     cfg.config_files.emplace_back(std::move(cfg.config_file_path));
-  for (auto& file : cfg.config_files)
+  for (const auto& file : system::loaded_config_files())
     VAST_INFO("loaded configuration file: {}", file);
```

Comment:
```suggestion
    VAST_INFO("loaded system configuration file: {}", file);
```

Reply:
I've instead removed this; turns out that we printed this twice otherwise.